static/components/ComponentArray.js

```diff
@@ -237,11 +232,6 @@ class ArrayComponent extends TextFieldComponent{
     console.log("this.value",this.value);
     // debugger;
```

Comment:
I know this isn't your code but it would be a good idea to remove all commented code and console statements from the code.  If we need to reference deleted code, we can look at the file history in git.  It's just a code hygiene thing.  One big question everyone reading the code will have is "why is this code commented out?"

Reply:
Is on my todo list actually - exactly as you've said, to remove all unused code and logger/console lines.  Wanted to get more of the DB checked over first before doing that though.

Reply:
I'll make an issue for this one as well, again so that people know it's coming.